.vscode/settings.json

```diff
@@ -1,8 +0,0 @@
-{
-  "workbench.colorCustomizations": {
-    "titleBar.activeBackground": "#00da63", // change this color!
-    "titleBar.inactiveBackground": "#00da63", // change this color!
-    "titleBar.activeForeground": "#ffffff", // change this color!
-    "titleBar.inactiveForeground": "#ffffff" // change this color!
-  }
-}
```

Comment:
This file should never have been commited on `master`. It is in the `.gitignore`